test/integration/bad/removed_features/stanc.expected

```diff
@@ -1,240 +1,51 @@
   $ ../../../../../install/default/bin/stanc abs-deprecate.stan
-Error in 'abs-deprecate.stan', line 3, column 7: fabs was removed in Stan
-    2.33. Use abs instead. This can be automatically changed using the
-    canonicalize flag for stanc
+Semantic error in 'abs-deprecate.stan', line 3, column 7 to column 17:
+   -------------------------------------------------
+     1:  transformed data {
+     2:    real mu;
+     3:    mu = fabs(-1.2);
+                ^
+     4:  }
+     5:  parameters {
+   -------------------------------------------------
+
+A returning function was expected but an undeclared identifier 'fabs' was supplied.
+A similar known identifier is 'abs'
```

Comment:
Isn't `test/integration/bad/removed_features/*` supposed to be testing `src/frontend/Deprecation_removals.ml`? Right now most of these hit typecheck errors that don't know anything about "removed features."

Reply:
My intent was that they would start as tests for this, but going forward would also be useful to track what the error becomes after they're out of `Deprecation_removals`. I think we can eventually ditch them, but for now I don't see any harm